client/scripts/controllers/server/comments.ts

```diff
@@ -156,7 +156,7 @@ class CommentsController {
         'chain': comment.chain,
         'community': comment.community,
         'body': encodeURIComponent(newBody),
-        'version_history': versionHistory,
+        'new_version_history': !!body,
```

Comment:
I think `new_version_history` should be computed on the backend as well. No need to split up this logic across the backend and frontend when 'has the body been updated?' is a question that the backend is fully capable of answering.

Reply:
👍